code/modules/mob/living/brain/brain_item.dm

```diff
@@ -205,11 +205,10 @@
 		owner.death()
 		brain_death = TRUE
 
-/obj/item/organ/brain/process()	//needs to run in life AND death
-	..()
+/obj/item/organ/brain/check_damage_thresholds(mob/M)
+	. = ..()
 	//if we're not more injured than before, return without gambling for a trauma
 	if(damage <= prev_damage)
-		prev_damage = damage
```

Comment:
this should probably be kept, or healing brain damage would prevent trauma generation until you reach your previous highest damage point

Reply:
I was going to mention this earlier but I forgot. `check_damage_thresholds` is called by `applyOrganDamage` which does the same thing right after.

https://github.com/tgstation/tgstation/blob/6aab8e1be64a257c97a7bef80d31f021a000b58a/code/modules/surgery/organs/organ_internal.dm#L149-L150